From 622b9c1baa625a2a137863600e343d350ed411da Mon Sep 17 00:00:00 2001
From: Pan Ortiz <panjeoc@proton.me>
Date: Wed, 13 Nov 2024 13:12:14 +0000
Subject: [PATCH 50/56] arch: arm64: boot: dts: qcom: sm8150-xiaomi-nabu.dts:
 add slpi remoteproc to nabu dts This is needed to power the Sensors Low Power
 Island

---
 arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
index 064618319f20..db137bb583fd 100644
--- a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
+++ b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
@@ -30,6 +30,7 @@
 /delete-node/ &spss_mem;
 /delete-node/ &cdsp_mem;
 
+
 / {
 	model = "Xiaomi Pad 5";
 	compatible = "xiaomi,nabu", "qcom,sm8150";
@@ -809,6 +810,11 @@ &remoteproc_mpss {
 	firmware-name = "qcom/sm8150/xiaomi/nabu/modem.mbn";
 };
 
+&remoteproc_slpi {
+ status = "okay";
+ firmware-name = "qcom/sm8150/xiaomi/nabu/slpi_nb.mbn";
+};
+
 &sound {
 	compatible = "qcom,sm8150-sndcard";
 	pinctrl-0 = <&quat_tdm_active
-- 
2.48.1

